src/main/java/fun/epak/pak/controller/ErrorController.java

```diff
@@ -0,0 +1,15 @@
+package fun.epak.pak.controller;
+
+import fun.epak.pak.exceptions.SaveFileException;
+import org.apache.tomcat.util.http.fileupload.impl.FileSizeLimitExceededException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class ErrorController {
+
+    @ExceptionHandler({SaveFileException.class, FileSizeLimitExceededException.class})
+    public String getFileEror(){
```

Comment:
Typo `Eror` -> `Error` ale nazwalbym to sensowniej np `getErrorView`

Reply:
poprawione :)